.github/workflows/prepare-release.yml

```diff
@@ -13,3 +13,34 @@ jobs:
         uses: actions/checkout@v2
         with:
           submodules: false
+
+      - name: Update Changelog
+        id: update-changelog
+        uses: realm/ci-actions/update-changelog@dac063f8ebd86c201590d07536d6f56f86f66ba9
+        with:
+          changelog: ${{ github.workspace }}/CHANGELOG.md
+
+      - name: Update pubspec.yaml
+        uses: jacobtomlinson/gha-find-replace@b76729678e8d52dadb12e0e16454a93e301a919d #! 2.0.0
+        with:
+          find: 'version: .*'
+          replace: 'version: ${{ steps.update-changelog.outputs.new-version }}+alpha'
+          include: '**/pubspec.yaml'
+
+      - name: Update realm.podspec
+        uses: jacobtomlinson/gha-find-replace@b76729678e8d52dadb12e0e16454a93e301a919d #! 2.0.0
```

Comment:
Why do we need this specific commit b767296 of jacobtomlinson/gha-find-replace?

Reply:
This is a best practice when using 3rd party actions. If we were to use `jacobtomlinson/gha-find-replace@v2`, an attacker that compromises the `jacobtomlinson/gha-find-replace` repo can push the v2 tag pointing to a malicious commit. It's part of the security hardening recommendations by Github: https://docs.github.com/en/actions/security-guides/security-hardening-for-github-actions#using-third-party-actions.